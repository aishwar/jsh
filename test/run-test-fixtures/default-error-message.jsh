
## Custom error message
cd dir-that-does-not-exist

## Log message that you should never reach
echo Done