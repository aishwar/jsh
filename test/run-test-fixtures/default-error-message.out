[LOG]  Custom error message
[ERROR]  Command "cd dir-that-does-not-exist" failed execution with a return value of: 1
Program terminated due to error
